spec/models/comment_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Comment do
+  newperson = Users.create(name: 'John Connor', photo: 'www.google.com', bio: 'Hello my name is John')
+  newpost = Post.create(title: 'Sunday', text: 'Sunny Sunday')
+
+  subject { Comment.new(author_id: newperson.id, post_id: newpost.id, Text: 'Nice picture!') }
+  before { subject.save }
+
+  it 'author_id should be = 1' do
+    newperson.id = 1
+  end
+
+  it 'post_id should be = 1' do
+    newpost.id = 1
+  end
+
+  it 'Text should be Nice picture!' do
+    subject.Text = 'Nice picture!'
+  end
```

Comment:
* According to the project [requirements ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.) 

> Add unit specs for all of your models' methods and validations.
* Kindly add unit specs for the `update_comments_counter` method